lib/app/components/molecules/gender_select_checkbox.dart

```diff
@@ -0,0 +1,31 @@
+import 'package:flutter/material.dart';
+import 'package:favoritism_communication/app/services/gender_service.dart';
+import 'package:favoritism_communication/app/components/atoms/custom_checkbox.dart';
+import 'package:get/get.dart';
+
+class GenderSelectCheckbox extends StatelessWidget{
+  const GenderSelectCheckbox({
+    super.key,
+    this.onChanged,
+    this.values,
+  });
+
+  final Map<Gender, bool>? values;
+  final void Function(Gender? gender, bool? value)? onChanged;
+
+  @override
+  Widget build(BuildContext context){
+    return Row(
+      children: Gender.values.map((gender){
+        return SizedBox(
+          width: context.width / Gender.values.length,
+          child: CustomCheckbox(
+            label: gender.label,
+            value: values?[gender] ?? false,
+            onChanged: (value) => onChanged?.call(gender, value),
+          ),
+        );
+      }).toList(),
+    );
+  }
+}
```

Comment:
```suggestion
}

class _CustomCheckbox extends StatelessWidget {
  const _CustomCheckbox({
    this.label,
    this.onChanged,
    this.value = false,
  });

  final String? label;
  final void Function(bool? value)? onChanged;
  final bool value;

  @override
  Widget build(BuildContext context) {
    final Widget? title =
        label != null ? Text(label!, style: TextStyle(fontSize: 14)) : null;

    return CheckboxListTile(
      contentPadding: EdgeInsets.zero,
      value: value,
      selectedTileColor: Colors.red,
      onChanged: onChanged,
      title: title,
      controlAffinity: ListTileControlAffinity.leading,
    );
  }
}
```
`CustomCheckbox`を使いまわさない気がしたので、
`_CustomCheckbox`とプライベートクラスにしました。
プライベートクラスにしたらファイル削除お願いします